### PR TITLE
fix(secrets): Syntax for secret file references

### PR DESCRIPTION
**Pull request by german-muzquiz**
Updated documentation to reference secret files.

### CODE REVIEW

File: reference/halyard/secrets/index.md [hunk 1]
```diff
@@ -11,11 +11,16 @@ Storing Spinnaker configs in a git repository is a great solution for maintainin
 
 
 ## Secret Format
-To reference secrets in configs, use the following general format:
+To reference secrets in configs, use the following general format for secret literal values, like passwords and tokens:
 
 ```
 encrypted:<secret engine>!<key1>:<value1>!<key2>:<value2>!...
 ```
+While to reference secret files like kubeconfig files the syntax is:
```
Comment (dorbin) [inline reference/halyard/secrets/index.md]:
```suggestion
To reference secret files like kubeconfig files the syntax is:
```

**Review by dorbin — COMMENTED** [review]
LGTM, but @mattsanta can you take a look?

**Comment by dorbin** [comment]
Scratch that, @mattsanta . @plumpy instead.

**Review by aimeeu — APPROVED** [review]
LGTM

**Review by plumpy — APPROVED** [review]

**Pull request merged**